packages/zsteg/PKGBUILD

```diff
@@ -0,0 +1,43 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=zsteg
+_gemname=zsteg
+pkgver=v0.2.11.r0.g93cad73
+pkgrel=1
+pkgdesc='Detect stegano-hidden data in PNG and BMP.'
+arch=('any')
+groups=('blackarch' 'blackarch-stego' 'blackarch-anti-forensic')
+url='https://github.com/zed-0xff/zsteg'
+license=('MIT')
+depends=('ruby' 'ruby-zpng' 'ruby-iostruct')
```

Comment:
```suggestion
depends=('git' 'ruby' 'ruby-zpng' 'ruby-iostruct')
```

Reply:
You are also not packaging ruby-prime: cf. https://github.com/zed-0xff/zsteg/blob/93cad73375a99ef06676243f6d70e0d931e1d8c7/Gemfile#L5

Reply:
@noraj Is it better if we insert `git` in `makedepends`?